test/Microsoft.ML.Tests/Scenarios/Api/CookbookSamples/CookbookSamples.cs

```diff
@@ -467,7 +467,7 @@ private void TextFeaturizationOn(string dataPath)
                     BagOfBigrams: r.Message.NormalizeText().ToBagofHashedWords(ngramLength: 2, allLengths: false),
 
                     // NLP pipeline 3: bag of tri-character sequences with TF-IDF weighting.
-                    BagOfTrichar: r.Message.TokenizeIntoCharacters().ToNgrams(ngramLength: 3, weighting: NgramExtractingEstimator.WeightingCriteria.TfIdf),
+                    BagOfTrichar: r.Message.TokenizeIntoCharacters().ProduceNgrams(ngramLength: 3, weighting: NgramExtractingEstimator.WeightingCriteria.TfIdf),
```

Comment:
>ProduceNgrams [](start = 69, length = 13)

Don't forget to update cookbook.md!

Reply:
There is not such an example. Maybe the author didn't add it because it's static.

---
In reply to: [264406110](https://github.com/dotnet/machinelearning/pull/2898#discussion_r264406110) [](ancestors = 264406110)